src/badguy/rcrystallo.hpp

```diff
@@ -23,6 +23,7 @@ class RCrystallo final : public WalkingBadguy
 {
 public:
   RCrystallo(const ReaderMapping& reader);
+  RCrystallo(Vector pos, Vector start_pos, float vel_x, std::unique_ptr<Sprite> sprite, Direction dir, float radius, std::string& script, bool fall = false);
```

Comment:
`const Vector&`, `const std::string&`. Parameter list can be split in multiple lines (same for the definition in the `.cpp` file).